### PR TITLE
Add typings for reactstrap

### PR DESCRIPTION
**Pull request by mfal**
- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `npm run new-package package-name`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

**Comment by mfal** [comment]
Hey @alihammad!
FYI: Yet another PR. The other one was not based on the current master.

File: reactstrap/lib/ButtonGroup.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/// <reference types='react' />
```
Comment (andy-ms) [inline reactstrap/lib/ButtonGroup.d.ts]: You don't need to do this in every file, just in `index.d.ts`.

Comment (mfal) [inline reactstrap/lib/ButtonGroup.d.ts]: Ah... that's why the Bot is so noisy? Update is coming

Comment (andy-ms) [inline reactstrap/lib/ButtonGroup.d.ts]: No, the bot is just always noisy...

**Comment by andy-ms** [comment]
Thanks!

**Pull request merged**